cookbooks/env.rst

```diff
@@ -203,6 +203,17 @@ will evaluate to an empty string like any other unset variable:
     When several routes match a rule, the first one wins, the user order is
     kept. There's no preference regarding protocols.
 
+  .. tip::
+
+    For complex multi-applications projects where several applications are
```

Comment:
I would remove complex as this is true for any multi-app project, right?

Reply:
Well, I would say 50/50 so you're right, let's remove "complex"